proposals/0250-NextRpcIndication.md

```diff
@@ -96,6 +96,87 @@ Add the struct to the `SoftButton` struct. This will cover Show/SetDisplayLayout
 </struct>
  ```
 
+### iOS
+1. We're adding in a public manager-level enum
+```ObjectiveC
+typedef NS_ENUM(NSInteger, SDLNextFunction) {
+        Default =0, 
+        PerformChoiceSet, 
+        Alert, 
+        ScreenUpdate, 
+        Speak, 
+        AccessMicrophone, 
+        ScrollableMessage, 
+        Slider, 
+        SendLocation, 
+        DialNumber, 
+        OpenMenu 
+};
+```
+
+2. Add a manager-level class `NextFunctionInfo` with an initializer 
+
+NextFunctionInfo.h
+```ObjectiveC
+@interface NextFunctionInfo: NSObject 
+	@property (copy, nonatomic, readonly) SDLNextFunction *nextFunction;
+	@property (copy, nonatomic, readonly, nullable) NSString *loadingText;
+@end
+```
+
+NextFunctionInfo.m
+```ObjectiveC
+@implementation NextFunctionInfo
```

Comment:
This implementation code is unnecessary. Only non-obvious or implementation code that could be implemented in several different ways needs to be added.